module/x/gravity/keeper/batch_test.go

```diff
@@ -98,6 +98,26 @@ func TestBatches(t *testing.T) {
 		assert.Equal(t, expFirstBatch.Transactions[i], gotFirstBatch.Transactions[i].ToExternal())
 	}
 
+	// persist confirmations for first batch
```

Comment:
this is good test code. But I wonder if we should be continuing to expand this batches test or splitting off to smaller tests.